libs/ConnectivityClient.py

```diff
@@ -1,28 +1,37 @@
 import paramiko
+import paramiko.ssh_exception
+import time
 from prometheus_client import Counter, Histogram
 from libs.TimeRecorder import TimeRecorder
 from libs.PrometheusExporter import CommandTypes, LabelNames
+
 from libs.loggerClass import Logger
 
 
 class MetricLabels:
-    STATUS_CODE = 'status_code'
-    HOST = 'host'
-    ENDPOINT = 'endpoint'
+    STATUS_CODE = "status_code"
+    HOST = "host"
+    ENDPOINT = "endpoint"
+
 
 class ResultStatusCodes:
```

Comment:
you can use the http lib for that 

example:
from http import HTTPStatus

HTTPStatus.OK == 200
>>> True

There is no flag as success afaik

Reply:
Again, @fraugabel can probably elaborate as this is something that came with rebase.